### DISCUSSION TITLE
Linux agent for asahi Linux - Which one ?

### DISCUSSION THREAD

**Discussion by tootai** [Ylianst/MeshCentral#5624] [Q&A]
Hello,
I'm running Debian 12 on Asahi Linux Apple M1 and can't get any executable Linux arm64 version  running on this machine. Did I do something wrong or there is no binary for Asahi Linux ?

Daniel

**Comment by si458** [comment]
The is a bug with arm64 #5108
The bug has been fixed but we need to build new agents and release them etc

**Comment by tootai** [comment]
Hello. Any news about M[1|2|3] machines ? I just tried again with Linux ARM64-HF as well as ARM64 bit (glibc/2.24 NOKVM) no more luck.

Also, in Pop up launched for install I see 3 buttons, translated in FR: 'Installer', 'Se' and 'Annuler'. What does 'Se' mean ?

Install I get undefined
Se I get denied

this for both tested versions.

**Reply by si458** [reply]
I have an M1 mac now hut I don't wanna loose the OS it's on cuz I'm using it for my fixing of mac bugs, if the any other OS/hardware where its not working on arm64?

**Reply by tootai** [reply]
The same M1 has meshcentral installed on MacOS and it's OK. Don't know about other ARM hardware
